### PR TITLE
Implement account page

### PR DESCRIPTION
**Pull request by Jonas22rr**

### CODE REVIEW

File: worktime-tracker-dev/components/account/ResetPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import AccountInput from "./Input";
+
+export default function resetPassword() {
+    const [inputValue, setInputValue] = useState<string>("");
```
Comment (D3nn7) [inline worktime-tracker-dev/components/account/ResetPassword.tsx:5]: React Hook "useState" is called in function "resetPassword" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

**Review by D3nn7 — CHANGES_REQUESTED** [review]
Please check the SonarCloud information about your code quality and fix the code.

**Comment by D3nn7** [comment]
You can also install the SonarCloud plugin for VS Code  to see the tips,... a bit earlier.

File: worktime-tracker-dev/components/account/ResetPassword.tsx (hunk 1 above)
Comment (Jonas22rr) [reply worktime-tracker-dev/components/account/ResetPassword.tsx:5]: revised

**Comment by Jonas22rr** [comment]
Please review the code and approve the pull-request.
The code smells from sonar cloud can not be fixed.

**Pull request merged**